src/expr/src/scalar/func.rs

```diff
@@ -2518,6 +2533,8 @@ impl BinaryFunc {
             BinaryFunc::PowerNumeric => power_numeric(a, b),
             BinaryFunc::RepeatString => repeat_string(a, b, temp_storage),
             BinaryFunc::GetByte => get_byte(a, b),
+            BinaryFunc::ConstantTimeEqBytes => constant_time_compare_bytes(a, b),
+            BinaryFunc::ConstantTimeEqStrings => constant_time_compare_strings(a, b),
```

Comment:
Best to keep the names in sync as much as possible.

```suggestion
            BinaryFunc::ConstantTimeEqBytes => constant_time_eq_bytes(a, b),
            BinaryFunc::ConstantTimeEqStrings => constant_time_eq_strings(a, b),
```